Core/src/main/java/com/plotsquared/core/util/Permissions.java

```diff
@@ -87,12 +88,12 @@ public static boolean hasPermission(PlotPlayer<?> player, String permission) {
      * @param permission
      * @return
      */
-    public static boolean hasPermission(CommandCaller caller, String permission) {
+    public static boolean hasPermission(PermissionHolder caller, String permission) {
         if (caller.hasPermission(permission)) {
             return true;
-        } else if (caller.isPermissionSet(permission)) {
```

Comment:
I don’t know, but I also don’t understand why this was here in the first place. That worries me a little :p

Reply:
I think I do:
Imagine someone having the admin permission, but still have their specific permission set to deny.
We have some teammembers which might have all moderator permissions except the ban permissions, because we know they like to play thor.

In this case, the permission is false but is set, so we exit early without checking for the admin permission and what happens next. Otherwise a permission set to `DENY` would be mistreated.

Reply:
I don't think we should deal with that, that's up to the permission plugin to handle.

Reply:
You _can't_ in this case. If the permission is `DENY`, `hasPermission` will return false, which doesn't stop PS from checking wildcards and admin permissions. That's why there is an explicit `isPermissionSet` API.

Reply:
Right, but this is not API offered by Vault, so then we'd get different behaviour depending on if we are dealing with offline vs. online players? That doesn't feel awesome.